api/all.yaml

```diff
@@ -2145,6 +2164,41 @@ components:
         - PATH_CREATE_TABLE
         - PATH_REFRESH
       type: string
+    GetMetastoreSummaryResponse:
+      type: object
+      properties:
+        metastore_id:
+          description: Unique identifier for the metastore.
+          type: string
+        name:
+          description: User-specified name of the metastore.
+          type: string
+        storage_root_credential_id:
+          description: UUID of storage credential to access the metastore storage_root.
+          type: string
+        cloud:
+          description: The cloud vendor of the metastore.
+          type: string
+        region:
+          description: The cloud region of the metastore.
+          type: string
+        global_metastore_id:
+          description: Globally unique metastore ID across clouds and regions, of the form `cloud:region:metastore_id`.
+          type: string
```

Comment:
Are these fields needed at this time?

Reply:
We only need the metastore id at this point, should I just keep that for now?